src/Microsoft.TemplateEngine.Cli/Microsoft.TemplateEngine.Cli.csproj

```diff
@@ -34,6 +34,11 @@
     <PackageReference Include="Wcwidth.Sources" ExcludeAssets="all" GeneratePathProperty="true" />
   </ItemGroup>
 
+  <!-- forces the version in sub dependencies -->
+  <ItemGroup>
+    <PackageReference Include="System.Net.Http" Version="4.3.4" />
```

Comment:
Out of curiosity, which dependency was referencing `System.Net.Http`?

Reply:
Old parser

Reply:
We need to remember to remove it when we migrate to System.CommandLine